spec.html

```diff
@@ -56,13 +60,12 @@ <h3>Basic Types</h3>
               <h3>Non-terminals</h3>
 
               <p>The following specifies the rest of the BSON
-              grammar. Note that quoted strings represent terminals,
-              and should be interpreted with C semantics
-              (e.g. <code>"\x01"</code> represents the byte <code>0000
-              0001</code>). Also note that we use the <code>*</code>
-              operator as shorthand for repetition
-              (e.g. <code>("\x01"*2)</code>
-              is <code>"\x01\x01"</code>). When used as a unary
+              grammar. We use the notation <code>signed_byte(n)</code>
```

Comment:
Looking at this again, it seems wrong for this content about the signed_byte terminal to be in the section for non-terminals. I'd delete this content from here and move it up to be part of the the table titled "Basic Types". Presumably it would be listed adjacent to `byte` and `null_byte`.